Automattic/pocket-casts-android
# Enable sponsored for everyone

## Description
Enables sponsored podcasts for everyone on the discover screen.

## Testing Instructions
1. Create a build of the app where you can purchase Plus
2. Open the app and create an account with Plus
3. Go to the discover page
4. See the sponsored podcast blocks

## Checklist
